tools/sandbox_e2e_smoke_test.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+import sys
+import multiprocessing
+from pathlib import Path
+
+project_root = Path(__file__).parent.parent
+for python_path in [str(project_root), str(project_root / "src")]:
+    sys.path.append(python_path)
```

Comment:
Let's start with a quick'n'dirty `sys.path` update, just to be sure that both the git repo's root (we need this to be able to do `from sandbox.basic import app`) and the "src/" folders are part of the Python path :-)